Components/HeroPostHeader/Partials/Categories/index.twig

```diff
@@ -0,0 +1,10 @@
+{% if post.categories %}
+  <div class="categories">
+    <span class="categories-label">Published in </span>
+    {% for category in post.categories %}
+      <span class="categories-category">
+        <a href="{{ category.link }}" class="categories-categoryLink">{{ category.name }}</a>,
```

Comment:
can't add the comma with CSS so that the last item also does not have it? Also then shouldn't it be a list?